### DISCUSSION TITLE
Supporting Markdown Alerts

### DISCUSSION THREAD

**Discussion by markwinter** [scalar/scalar#3402] [Ideas]
I was wondering whether it would be possible to support Markdown Alerts or not? I'd like to add them to the description fields to call out certain details.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

**Comment by antlio** [comment]
hey @markwinter, that's a great idea, working on it!

**Reply by provinzkraut** [reply]
@antlio Is this still being worked on? Couldn't find an open issue / PR. This would be a really nice feature to have! :)

**Reply by antlio** [reply]
hey @provinzkraut! i did work on it but never push the code somehow! will keep you update it once done

**Comment by antlio** [comment]
@markwinter @provinzkraut the PR has now been open regarding the alert support #4918 using the same supported format as GitHub.

**Reply by provinzkraut** [reply]
Awesome, thanks a bunch! :)
